source/fuzz/fuzzer_pass_split_blocks.cpp

```diff
@@ -50,39 +51,51 @@ void FuzzerPassSplitBlocks::Apply() {
       continue;
     }
     // We are going to try to split this block.  We now need to choose where
-    // to split it.  We do this by finding a base instruction that has a
-    // result id, and an offset from that base instruction.  We would like
-    // offsets to be as small as possible and ideally 0 - we only need offsets
-    // because not all instructions can be identified by a result id (e.g.
-    // OpStore instructions cannot).
-    std::vector<std::pair<uint32_t, uint32_t>> base_offset_pairs;
+    // to split it.  We describe the instruction before which we would like to
+    // split a block via the opcode 'opc' of the relevant instruction, a base
+    // instruction 'base' that has a result id, and the number of instructions
+    // with opcode 'opc' that we should skip when searching from 'base' for the
+    // desired instruction.  (When the instruction before which we would like to
+    // split the block actually has a result id, the instruction is used as
+    // base, 'opc' is its opcode, and there are 0 instructions to skip.)
+    std::vector<std::tuple<uint32_t, SpvOp, uint32_t>> base_opcode_skip_triples;
```

Comment:
If we plan to change this later anyway, then never mind, But otherwise, could we just use `InstructionDescriptor`s here.

Reply:
Wow, of course!!